examples/test.js

```diff
@@ -0,0 +1,60 @@
+const passport = require("passport");
+const OutlookStrategy = require("../lib/strategy");
+
+passport.use(
+  new OutlookStrategy(
+    {
+      callbackURL: "http://localhost:3000/auth/outlook/redirect",
+      passReqToCallback: true,
+      clientID: "3d1dc301-4f26-4b9c-a641-640d237c5ebe",
+      clientSecret: "6o9sdc.u3eI-YQ_BGa4zeJ~G9_0mgbr2Tc",
```

Comment:
If these are personal credentials and not provided by Microsoft you should redact them right away and set this using environment variables, perhaps adding a comment as such to the test.

Reply:
Yes thanks! I will change it soon.